resources/PhoneNumberMetadataForTesting.xml

```diff
@@ -1065,14 +1065,13 @@
     <!-- For testing purposes, numbers starting with 24 are not considered US numbers.-->
     <territory id="US" countryCode="1" internationalPrefix="011"
                preferredExtnPrefix=" extn. " nationalPrefix="1"
-               mainCountryForCode="true" nationalPrefixOptionalWhenFormatting="true"
-               mobileNumberPortableRegion="true">
+               mainCountryForCode="true" mobileNumberPortableRegion="true">
       <availableFormats>
         <numberFormat pattern="(\d{3})(\d{4})">
           <format>$1 $2</format>
           <intlFormat>NA</intlFormat>
        </numberFormat>
-        <numberFormat pattern="(\d{3})(\d{3})(\d{4})">
+        <numberFormat pattern="(\d{3})(\d{3})(\d{4})" nationalPrefixOptionalWhenFormatting="true">
```

Comment:
Are we sure this is the only affected region?

Reply:
This is metadata "for testing", so I changed it so the unit tests pass

Reply:
fair enough - I'll get round to regenerating this at some point.